developers/weaviate/current/configuration/authentication.md

```diff
@@ -3,7 +3,7 @@ layout: layout-documentation
 solution: weaviate
 sub-menu: Configuration
 title: Authentication
-intro: By default, Weaviate runs without any form of authentication. To run Weaviate with authentication, you can configure OpenID authentication in the configuration file of Weaviate. Then, use a Bearer token to authenticate.
+intro: By default, Weaviate runs without any form of authentication. To run Weaviate with authentication, you can enable OpenID authentication in Weaviate's configuration file. Then, use a Bearer token to authenticate.
```

Comment:
Is this true? Below it says "If anonymous access is selected, Weaviate will accept requests...."

Reply:
Hmm. Okay maybe it could be clearer. I have re-phrased the latter part to read:

`By default, Weaviate is configured to accept anonymous...` and edited the line preceding the `AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED` environment variable to read `Anonymous access can be enabled or disabled in the Docker Compose yaml using the environment variable shown below`.

Reply:
Reads great, but it still has "Only a special group can write or delete resources". We should hide this until groups are actually supported

Reply:
Oh I see, yeah that's a good point. My understanding from the authorization docs is that Weaviate can give a subset of authenticated users "admin" rights, and "read-only" rights to the other authenticated users. And then there is a third set of unauthenticated (anonymous) users. 

This doesn't rely on OIDC "groups" though, so I have rephrased this to say "Only a special subset of admin users can write or delete resources.".

Reply:
Ok cool! Could you double check that those groups are supported? Not sure who would know that

Reply:
Sure! Will review this separately as this has implications for both AuthZ and AuthN pages.